newIDE/app/src/EventsSheet/index.js

```diff
@@ -276,6 +276,18 @@ export default class EventsSheet extends React.Component<Props, State> {
     });
   };
 
+  _selectionCanReceiveSubEvents = () => {
+    return getSelectedEvents(this.state.selection).some(event => {
+      return event.canHaveSubEvents();
+    });
+  };
+
+  _selectionCanToggleDisabled = () => {
+    return getSelectedEvents(this.state.selection).some(event => {
+      return event.getType() !== 'BuiltinCommonInstructions::Comment';
```

Comment:
Not bad, but we can do a bit better: `return event.isExecutable()` :)

Reply:
ah nice. I didn't notice this method initially. Will amend this now